src/pytorch_lightning/utilities/migration/migration.py

```diff
@@ -47,6 +47,7 @@ def _migration_index() -> Dict[str, List[Callable[[_CHECKPOINT], _CHECKPOINT]]]:
         "1.6.0": [_migrate_loop_global_step_to_progress_tracking, _migrate_loop_current_epoch_to_progress_tracking],
         "1.6.5": [_migrate_loop_batches_that_stepped],
         "1.9.0": [_migrate_model_checkpoint_save_on_train_epoch_end_default],
+        "2.0.0": [_drop_apex_amp_state_from_checkpoint],
```

Comment:
```suggestion
        "2.0.0": [_drop_apex_amp_state_from_checkpoint],
```
technically "from_checkpoint" is redundant for any of the migration names

Reply:
```suggestion
        "2.0.0": [_drop_apex_amp_state],
```